### PR TITLE
Adapt the plugin to nodebb v3, fix bugs and style, and add localization

### PR DESCRIPTION
**Pull request by ShlomoCode**

### CODE REVIEW

**Comment by ShlomoCode** [comment]
It should be noted that I was unable to get the plugin to work in the Persona theme, only in Harmony. I can't figure out why, it doesn't seem to load for some reason. @barisusakli maybe you understand why?
CleanShot 2023-05-10 at 18 48 06@2x

**Comment by barisusakli** [comment]
For this plugin to work the reactions plugin must be loaded after the theme? You can check the plugin order inthe acp plugins page. The reactions.tpl form this plugin should overwrite the one in the theme.

**Comment by ShlomoCode** [comment]
@barisusakli Nice, you were right!
The thing is simply that as soon as I change the theme in /admin/appearance/themes then the theme I chose jumps to be the last one in the loading list.
After manually changing the loading order also works in Persona

**Comment by barisusakli** [comment]
Hmm maybe activating a theme should put it in the first order @julianlam thoughts?

File: less/reactions.less [hunk 1]
```diff
@@ -32,7 +32,7 @@
         cursor: pointer;
 
         &.max-reactions {
-            display: none;
+            display: none !important;
```
Comment (barisusakli) [inline less/reactions.less]: 3.0 no longer uses less files so this file should be renamed to scss/reactions.scss and plugin.json should be updated to use the scss block

File: templates/admin/plugins/reactions.tpl [hunk 2]
```diff
@@ -26,6 +26,8 @@
...
 
 
-<button id="save" class="floating-button mdl-button mdl-js-button mdl-button--fab mdl-js-ripple-effect mdl-button--colored">
-    <i class="material-icons">save</i>
-</button>
+<div class="floating-button">
+	<button id="save" class="btn btn-primary position-fixed bottom-0 end-0 px-3 py-2 mb-4 me-4 rounded-circle fs-4" type="button" style="width: 64px; height: 64px;">
+		<i class="fa fa-fw fas fa-floppy-disk"></i>
+	</button>
+</div>
```
Comment (barisusakli) [inline templates/admin/plugins/reactions.tpl]: can use `<!-- IMPORT admin/partials/save_button.tpl -->` to use the core save button

File: less/reactions.less (hunk 1 above)
Comment (ShlomoCode) [reply less/reactions.less]: Hmmm... I don't know SCSS syntax, I converted with [this tool](https://jsonformatter.org/less-to-scss), hope it's ok 😄

Comment (barisusakli) [reply less/reactions.less]: They are mostly identical, there are some syntax differences related to mixins but they are not used here so you should be good.

Just build nodebb with the plugin active and if there are scss errors they will show up during build.

File: templates/admin/plugins/reactions.tpl (hunk 2 above)
Comment (ShlomoCode) [reply templates/admin/plugins/reactions.tpl]: done

File: less/reactions.less (hunk 1 above)
Comment (ShlomoCode) [reply less/reactions.less]: Yes, I checked and it was fine after the build

**Comment by barisusakli** [comment]
LGTM

**Comment by barisusakli** [comment]
published 2.0.0

**Comment by ShlomoCode** [comment]
@barisusakli By the way, I think you should delete https://github.com/barisusakli/nodebb-plugin-reactions, it was a bit confusing to understand what the official repo is :)

**Comment by barisusakli** [comment]
done

**Comment by ShlomoCode** [comment]
@barisusakli Do you understand why this appears to me when I install through the ACP? 🤔 
CleanShot 2023-05-11 at 01 45 06@2x
I wrote `"compatibility": "^3.0.0"` שnd I'm running nodebb 3.0.1

If I understand correctly, the ^ mark means that it is compatible with 3.x.x, right?

**Comment by barisusakli** [comment]
Yeah your "compatibility" block is correct that warning should go away when nodebbs start using 2.0.0 that have set their "submit usage data" to on.

**Pull request merged**